WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/AuthorsUseCase.kt

```diff
@@ -149,6 +153,16 @@ constructor(
         return items
     }
 
+    private fun getMaxViews(authors: List<Author>): Int {
```

Comment:
I'd rather use something like `domainModel.authors.maxBy { it.views }?.views ?: -1` which allows you to inline this method.

Reply:
Nice!  Will do.  TIL